integrated_channels/integrated_channel/constants.py

```diff
@@ -3,3 +3,4 @@
 """
 
 ISO_8601_DATE_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
+TASK_LOCK_EXPIRY_SECONDS = 43200
```

Comment:
last nit: I find `60 * 60 * 12` a little easier/quicker to evaluate as 12 hours (rather than remembering that 86400 seconds is one day and dividing that by 2).